mobile/.bazelrc

```diff
@@ -77,6 +77,12 @@ build:mobile-dbg-ios --config=ios
 # https://github.com/envoyproxy/envoy/issues/24977
 build:test-android --define=static_extension_registration=enabled
 
+# Default flags for iOS tests.
+build:test-ios --config=ios
+# Needed for running test Envoy servers as HTTP(S) proxies.
+build:test-ios --define=envoy_mobile_listener=enabled
+build:test-ios --define=static_extension_registration=enabled
```

Comment:
so we'd discussed having an auto-register listener library - can we do that instead of this?